cli/test/test_runner_kubernetes.py

```diff
@@ -0,0 +1,58 @@
+import os
+import unittest
+
+import utils as testutils
+
+from popper.config import PopperConfig
+from popper.parser import YMLWorkflow
+from popper.runner import WorkflowRunner
+
+from popper.cli import log as log
+
+
+class TestKubernetesDockerRunner(unittest.TestCase):
+    def setUp(self):
+        log.setLevel('CRITICAL')
+
+    @unittest.skipIf(
+        os.environ['ENGINE'] != 'docker' or os.environ['RESMAN'] != 'kubernetes',
+        'ENGINE != docker or RESMAN != kubernetes')
+    def test_docker_basic_run(self):
+        repo = testutils.mk_repo()
```

Comment:
once we derive from `PopperTest`, then this would become `self.mk_repo()`

Reply:
Done